domain/modeldefaults/state/state.go

```
@@ -249,20 +249,18 @@ func (s *State) ModelMetadataDefaults(
 		return nil, errors.Trace(err)
 	}
 
-	stmt := `
-SELECT m.name, ct.type
+	modelUUID := modelUUID{UUID: uuid.String()}
+	var result modelMetadata
+	stmt, err := s.Prepare(`
```

static analysis is crying because this error is not checked

---

just pushed a fix!